### PR TITLE
FIX format date

### PR DESCRIPTION
**Pull request by AuraAlba**
### Description
Format takes the local timezone so the dates shown depends on the country to fix that only the day, month and year are taken to the formatted string and avoid the bug

### Changes made
Add formatUTCdate function
Change format date-fns function
update tests

### How to test
Clone the Institution Portal MFE
Run npm install.
Run npm run start
Go to http://localhost:1980/classes

### CODE REVIEW

File: src/helpers/index.js [hunk 1]
```diff
@@ -1,5 +1,21 @@
 import { format } from 'date-fns';
 
+/**
+ * Format a UTC date
+ *
+ * @param {Date} startDate UTC date
```
Comment (01001110J) [inline src/helpers/index.js:6]: We're using a string and not a `date` object.

```suggestion
 * @param {string} date - UTC date
```

File: src/helpers/index.js [hunk 2]
```diff
@@ -1,5 +1,21 @@
...
+ * @param {string} formatStr pattern to be formatted
```
Comment (01001110J) [inline src/helpers/index.js:7]:
```suggestion
 * @param {string} formatStr - pattern to be formatted
```

File: src/helpers/index.js [hunk 3]
```diff
@@ -1,5 +1,21 @@
...
+ * @returns {string} Formatted date
+ */
+export const formatUTCDate = (date, formatStr = 'MM/dd/yy') => {
+  const [year, month, day] = date.substr(0, 10).split('-');
```
Comment (01001110J) [inline src/helpers/index.js:11]: Is possible to replace the `substr` for another method? I was checking the reference and I found this:

> This feature is no longer recommended
> Avoid using it, and update existing code if possible
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/substr

File: src/helpers/index.js [hunk 4]
```diff
@@ -1,5 +1,21 @@
 import { format } from 'date-fns';
 
+/**
+ * Format a UTC date
+ *
+ * @param {Date} startDate UTC date
+ * @param {string} formatStr pattern to be formatted
+ * @returns {string} Formatted date
+ */
+export const formatUTCDate = (date, formatStr = 'MM/dd/yy') => {
```
Comment (01001110J) [inline src/helpers/index.js:10]: What would happen if I pass `undefined` or `null` in `date?` Are you sure this won't blow up? :thinking:

**Review by 01001110J — APPROVED** [review]

**Review by anfbermudezme — APPROVED** [review]

**Pull request merged**